examples/cells_output.ts

```diff
@@ -31,7 +31,7 @@
 import * as helpers from "../auraescript/gen/helpers.ts";
 import * as runtime from "../auraescript/gen/cells.ts";
 
-let cells = new runtime.CellServiceClient();
+let cells = new runtime.CellServiceClient("./pki/ca.crt", "./pki/_signed.client.nova.crt", "./pki/client.nova.key", "127.0.0.1:8080");
```

Comment:
super nit: if the socket could be a socketaddress and we could take advantage of typescript's type safety that would be neat.  i don't know if that's even possible (there's an "ip-address" node module but i don't know if that's importable through typescript+deno or how it translates to a type in rust).

Reply:
Maybe it could even be a [Deno.Conn](https://deno.land/api@v1.31.0?s=Deno.Conn), I wonders if we can give tonic client an already open socket

Reply:
i _think_ i'm more comfortable passing data from TS to auraescript for it to open the connection, similarly to how `ae` would do it.  finding the balance between "rich" script and client is tricky but opening the connection crosses the line for me (but this is only my opinion).

Reply:
Looking a Deno 's op interface I don't think it is possible to transfer complex object from js to rust